arangod/Replication2/ReplicatedLog/LogLeader.cpp

```diff
@@ -312,6 +310,14 @@ void replicated_log::LogLeader::executeAppendEntriesRequests(
                           }
                           return {};
                         });
+                    if (responseResult.fail()) {
+                      LOG_TOPIC("a32de", DEBUG, Logger::REPLICATION2)
+                          << "appendEntries failed, follower already gone: "
+                          << responseResult.result();
+                    }
+
+                    auto [preparedRequests, resolvedPromises] =
+                        std::move(responseResult.get());
```

Comment:
I don't think it's okay to silently stop here - the leader will stop working in a situation where it is not expected to. I rather think we should explicitly ignore resigned errors, but continue to crash with everything else.

Reply:
Addressed in https://github.com/arangodb/arangodb/pull/20628/commits/5b019267229a77636c84ae0a2b0e65035f7c3c36